server/cluster/cluster.go

```diff
@@ -318,6 +398,18 @@ func (c *Cluster) loadSchemaLocked(ctx context.Context) (map[string]*clusterpb.S
 	return schemaMap, nil
 }
 
+func (c *Cluster) loadNodeLocked(ctx context.Context) (map[string]*clusterpb.Node, error) {
+	nodes, err := c.storage.ListNodes(ctx, c.clusterID)
+	if err != nil {
+		return nil, errors.Wrap(err, "clusters loadNodeLocked")
+	}
+	nodeMap := make(map[string]*clusterpb.Node, len(nodes))
```

Comment:
We had better avoid suffix a variable with its type because this is verbose. As for this case, `nameNodes` is preferred as it tells the mapping information: name -> node.